### DISCUSSION TITLE
Make a version of docker

### DISCUSSION THREAD

**Discussion by Meteor-showers** [BingChunMoLi/MoLiAPI#24] [Ideas]
Can you Make a version of docker. Is it more convenient

**Comment by Meteor-showers** [comment]
Good project ，refueling！

**Comment by BingChunMoLi** [comment]
Docker requires binding configuration and random img data

**Reply by Meteor-showers** [reply]
sure

**Comment by BingChunMoLi** [comment]
show 616f36b30519c3160d77a3cc401bc0afbb5fa74c  
branch in feature-h2

**Reply by Meteor-showers** [reply]
OK thanks

**Comment by BingChunMoLi** [comment]
docker hub is https://hub.docker.com/r/bingyantang/moliapi
this version is Alpha
yiyan data in 100
shici data in 100
data file in https://github.com/BingChunMoLi/MoLiAPI/blob/616f36b30519c3160d77a3cc401bc0afbb5fa74c/src/main/resources/db/data-h2.sql

**Reply by Meteor-showers** [reply]
perfect

**Comment by BingChunMoLi** [comment]
Please use the latest version docker run -d --name api -p 8090:8090 -v /.api/:/soft/config/ bingchunmoli/moliapi:latest
